cypress/e2e/links.spec.js

```diff
@@ -45,8 +45,8 @@ describe('test link marks', function() {
 				.type('google.com')
 
 			cy.getContent()
-				.find('a[href*="google.com"]')
-				.click({ force: true })
+				.get('a[href*="google.com"]')
```

Comment:
You should stick with `find` as `get` will not only search the editor content but the whole document.
Probably something like `getContent().find(...).should('not.exist')` would work

Reply:
Thanks for the tip, I am still a cypress noobie :)